src/SignalRServiceExtension/Bindings/SignalRData.cs

```diff
@@ -1,11 +1,14 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using Microsoft.Azure.SignalR;
+
 namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
 {
     internal class SignalRData
     {
         public string Target { get; set; }
         public object[] Arguments { get; set; }
+        public ServiceEndpoint[] Endpoints { get; set; }
```

Comment:
`ServiceEndpoint` is an class we defined in c#, but not recognized in non c# language, like js, python, java. Usually built-in types are safe to use, like string, int, arrays. Shall we make this string[] and do some further process later.